test/base_delayed_paperclip_test.rb

```diff
@@ -1,254 +0,0 @@
-###
-#  DEPRECATED, Don't Use
-###
-
-module BaseDelayedPaperclipTest
```

Comment:
@morgoth cool. you're right. If we're not running them on Travis then it's time we let them go an dmove into the future!